docs/src/developers/slab_sea_ice.jl

```diff
@@ -0,0 +1,212 @@
+# # Implementing a Slab Sea Ice Component
```

Comment:
This file slipped in, it was meant for another PR. I will delete it after the docs build so I don't need to wait that the docs build once again